### ISSUE TITLE
NFR: add DataDirector info to Class/ObjectBrick/FieldCollection Editor

### ISSUE THREAD

**Issue by kaurov** [BlackbitDigitalCommerce/pimcore-data-director#41]
Can you please implement the new feature in the next version?

All following editors have list of fields to edit:
Settings-> Data Object -> Classes
Settings-> Data Object -> ObjectBricks
Settings-> Data Object -> Fields-Collections

That is usual workplace for data manager. This is where he expects to controll if all fields are existing and imported properly.

Please display the following information in there for each field:
* Raw Data Field (SelectBox + callback function editor + data sample)
* xPath (just info, to see what means selected Raw Data Field)
* Max data length (just info, to set up Columnlength property)

**Comment by BlackbitNeueMedien** [comment]
Thanks for your suggestion but I have not understood it completely yet. 

> Raw Data Field (SelectBox + callback function editor + data sample) (each language)

This is already in attribute mapping panel, isn't it?

> Expression/xPath (just info, to see what means selected Raw Data Field)

You mean to show the XPath (or more generally the selector expression) for the raw data field? Currently the raw data field name and (if available) an example value from the import data gets shown in the raw data field select box. And there you want to additionally add the XPath? This would get rather long option labels in the raw data field select box...

> Max data length (just info, to set up Columnlength property)
> Ideally, to set automatically or to show the button to set 'Not editable = true' and columnlength = min(10, (Max data length)*1.3)
Here you man that you can edit the field definitions directly from the attribute mapping panel?

**Comment by kaurov** [comment]
No, I mean as written upper, the Clsss/ObjectBricks/FieldsCollections editor, not Datadirector tabs.

**Comment by BlackbitNeueMedien** [comment]
Ah I see. Currently there is no JS event for the field definitions in https://github.com/pimcore/pimcore/blob/11.x/bundles/AdminBundle/Resources/public/js/pimcore/events.js
I could only override the field type JS classes with `Ext.override()` but this won't work when there is another bundle which does the same (e.g. to change a Pimcore default field's behaviour).

But I understand the requirement: You want to have an overview for a certain class / object brick / field collection which dataports import to this. For single fields we already have this when you click on the field name in attribute mapping:
<img width="1348" alt="Bildschirmfoto 2022-07-25 um 20 47 27" src="https://user-images.githubusercontent.com/8749138/180851644-b97be5af-57fa-4618-9605-1a3410f28d90.png">
(Here we have 2 imports which import to field `Published` of class `Product` and 1 export which exports this field)

But we currently do not have a complete overview for the whole class. Will think about visualization of this. Best would indeed be if we could add it to the field definitions... will try to find a way. 
Of course, I will put this on the todo list but if anyone is willing to pay for this, this will be implemented much faster...
